### DISCUSSION TITLE
Sensor Timeout Error

### DISCUSSION THREAD

**Discussion by dnnybrns** [Security-Onion-Solutions/securityonion#5892] [Unsupported Versions] [answered]
I am building a distributed architecture and keep getting a timeout error for salt towards the end of the install for my sensor. I have noticed that the installation stays on 20% Accepting salt key at manager for quite some time, which I think may be the cause of my issue. Some of the fixes I've tried are the following:

-Was using version 2.3.70 so upgraded to 2.3.80

-Bypassed Cisco firewall

-Tried manually adding sensor to firewall rules on manager.

-Rebooted manager

**Comment by palichx** [comment, marked as answer]
Hi,
did you install OS update(before sun soup)? sound like manager and sensor have different salt version..
try to run _yum update_. and then _sudo salt-call state.highstate_

**Reply by dnnybrns** [reply]
Thanks. I will try this when I go in tomorrow, but will they have different salt versions if it's a completely fresh build? I deleted both VMs from disk and built both master and sensor from the new 2.3.80 iso

**Reply by palichx** [reply]
I thought you upgrade your sensors. yes, on fresh install salt will be same version. 
may be soremote password incorrect?

**Reply by dnnybrns** [reply]
My apologies for not being clear, but I did a fresh build. I could be wrong but it asks for the soremote password again if it is wrong. 

I've also built a sensor on similar system to see if I may have messed up somewhere and it successfully installs.

**Reply by dnnybrns** [reply]
Fixed it. I downloaded version 2.3.60 and it worked.
